src/pages/index.tsx

```diff
@@ -117,6 +118,11 @@ export function Pages() {
 
   const { isConnected } = useAccount();
 
+  /**
+   * Disconnects from the bundlr node if the chain or account changes.
+   */
+  useBundlrDisconnect();
+
```

Comment:
@solidoracle -- I'm not 100% on this, but I think you could attempt a separate branch with these 3 changes:

1. Place `useBundlrSession` here on line 124  (dont refactor into a separate hook)
2. Remove the Bundlr disconnect code from the `WalletDisconnectPage`, since using the session here should detect when an account is disconnected or changed and not needed on that component anymore.
3. Remove the `useBundlrSession` all from `embalm/index.tsx`, since its now being handled here (which is "higher" up the DOM than the embalmPage).

Essentially I think the core problem is that we want`useBundlrSession` at a more global level, so that it will handle account disconnects/changes outside of just the embalm process. Just moving up the hook (I believe) should accomplish this.